impeller/aiks/aiks_unittests.cc

```diff
@@ -2432,7 +2432,8 @@ TEST_P(AiksTest, ClipRectElidesNoOpClips) {
 
 TEST_P(AiksTest, ClearColorOptimizationDoesNotApplyForBackdropFilters) {
   Canvas canvas;
-  canvas.SaveLayer({}, std::nullopt,
+  Paint layer_paint = {.color = Color::BlackTransparent()};
```

Comment:
Hopefully this is just an incorrect test and we didn't have production code relying on the default parameter for Paint.

Reply:
Looking at some of the failed golden tests I'm thinking this may be too big of a breaking change.  We may have to switch Paint.color to an std::optional<Color> where it is interpreted as black or transparent black depending on the context.

Reply:
https://github.com/flutter/engine/pull/47026#discussion_r1365999875